packages/core/src/common/errors.ts

```diff
@@ -50,7 +50,9 @@ export const RADIOGROUP_RADIO_CHILDREN = `${ns} <RadioGroup> only supports <Radi
 export const RADIOGROUP_CHILDREN_OPTIONS_MUTEX =
     `${ns} <RadioGroup> children and options props are mutually exclusive.`;
 
-export const RANGESLIDER_NULL_VALUE = `${ns} <RangeSlider> value prop must be an array of two non-null numbers`;
+export const SLIDER_ZERO_STEP = `${ns} <Slider> stepSize cannot be zero.`;
```

Comment:
what backticks? this is a string for the console and we don't use backticks in any of the others.